src/store/metadata/stub/content/editions/beis_2005-2019_la_carbon_emission.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
so we need to have very specific file naming for the stubbed code to work. always has to be

`<dataset_identifier>_<edition_identifier>.json`, so in this instance `beis-2005-to-2019-local-authority-carbon-dioxide-co2-emissions_2005-2019.json`